pkgs/build-support/fetchgit/default.nix

```diff
@@ -90,6 +90,7 @@ stdenvNoCC.mkDerivation {
     ${netrcPhase}
     # required that git uses the netrc file
     mv {,.}netrc
+    export NETRC=$PWD/.netrc
```

Comment:
I think this is technically not needed because `$HOME/.netrc` will be used by default, but it seems better to be explicit.